osu.Server.Spectator/Hubs/Metadata/MetadataHub.cs

```diff
@@ -131,8 +131,13 @@ public async Task UpdateActivity(UserActivity? activity)
 
                 usage.Item.UserActivity = activity;
 
-                if (shouldBroadcastPresenceToOtherUsers(usage.Item))
-                    await broadcastUserPresenceUpdate(usage.Item.UserId, usage.Item.ToUserPresence());
+                await Task.WhenAll
+                (
+                    shouldBroadcastPresenceToOtherUsers(usage.Item)
+                        ? broadcastUserPresenceUpdate(usage.Item.UserId, usage.Item.ToUserPresence())
+                        : Task.CompletedTask,
+                    Clients.Caller.UserPresenceUpdated(usage.Item.UserId, usage.Item.ToUserPresence())
+                );
```

Comment:
I really prefer doing this as 2x`await` with explicit `if` for readability (I don't think performance concerns are valid for these kinds of usages?).

Reply:
In that case I'll also remove the one in broadcast, yes?

Reply:
The one below? Yeah I guess. The inline ternary if in here is specifically annoying to my eye.

Reply:
Let's leave this for now. In an IDE it's bearable legibility wise, just looks bad on github review.